src/submodule.c

```diff
@@ -688,6 +723,9 @@ int git_submodule_add_setup(
 		goto cleanup;
 	}
 
+	if ((error = can_add_submodule(repo, path)) < 0)
```

Comment:
There's a space missing in front of the first opening brace.

Reply:
I'm not sure what brace you are referring to here.

Reply:
Huh. Looks alright to me, too. Dunno either :P